test/common/common.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+function ensureKeys(keys, obj) {
+	for (var i = 0; i < keys.length; i++) {
+		if (!(keys[i] in obj) || keys[i] === undefined || keys[i] === null) {
+			console.warn('Required key: ' + keys[i] + ' is not found');
+			return false;
+		}
+	}
+	return true;
+}
+
+exports.ensureKeysNotNull = function(keys, obj) {
```

Comment:
If we `deepEqual` in the tests against fixtures, then these methods are probably not necessary anymore.